src/test/data/JsonAddressBookStorageTest/invalidAndValidClientAddressBook.json

```diff
@@ -19,5 +19,18 @@
     "clientPhone" : "9482424"
   }, {
     "clientPhone" : "948asdf2424"
-  } ]
+  } ],
```

Comment:
This json file should only contain valid and invalid clients. Others like orders and recipes can be left empty. Same goes for the other files. They should only contain what they say they are testing.

Can you help change all other files so that only one thing is tested each time?